src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -980,29 +1040,32 @@ private void DrawPackage(INugetPackage package, GUIStyle packageStyle, GUIStyle
                         }
                     }
 
-                    var versionComparison = installed.PackageVersion.CompareTo(package.PackageVersion);
-                    if (versionComparison < 0)
+                    if (currentTab == NugetWindowTab.UpdatesTab)
                     {
-                        // An older version is installed
-                        if (GUILayout.Button("Update"))
+                        var versionComparison = installed.PackageVersion.CompareTo(package.PackageVersion);
+                        if (versionComparison < 0)
                         {
-                            NugetPackageUpdater.Update(installed, package);
-                            UpdateInstalledPackages();
-                            UpdateUpdatePackages();
+                            // An older version is installed
+                            if (GUILayout.Button("Update"))
+                            {
+                                NugetPackageUpdater.Update(installed, package);
+                                UpdateInstalledPackages();
+                                UpdateUpdatePackages();
+                            }
                         }
-                    }
-                    else if (versionComparison > 0)
-                    {
-                        // A newer version is installed
-                        if (GUILayout.Button("Downgrade"))
+                        else if (versionComparison > 0)
                         {
-                            NugetPackageUpdater.Update(installed, package);
-                            UpdateInstalledPackages();
-                            UpdateUpdatePackages();
+                            // A newer version is installed
+                            if (GUILayout.Button("Downgrade"))
+                            {
+                                NugetPackageUpdater.Update(installed, package);
+                                UpdateInstalledPackages();
+                                UpdateUpdatePackages();
+                            }
                         }
                     }
 
-                    if (GUILayout.Button("Uninstall"))
+                    if (currentTab == NugetWindowTab.InstalledTab && GUILayout.Button("Uninstall"))
```

Comment:
Se above I would allow to uninstall package even on Update / Online tab. See Visual Studio package window. Or is the UI to bloated?